libs/vkd3d/device.c

```diff
@@ -2067,6 +2070,13 @@ static HRESULT vkd3d_init_device_caps(struct d3d12_device *device,
         physical_device_info->properties2.properties.sparseProperties.residencyNonResidentStrict = VK_FALSE;
     }
 
+    /* We need independent interpolation to use GPL. */
+    if (!physical_device_info->graphics_pipeline_library_properties.graphicsPipelineLibraryIndependentInterpolationDecoration)
```

Comment:
Is this required even if we compile FS and all vertex stages into one single library?

Spec wording kind of indicates that it is but I don't know if this is really the intent here, but I guess every driver that supports GPL will want to support this bit anyway.

Reply:
I looked for an exception, but there is none. Maybe it should be clarified in spec, but most likely drivers have a separate GPL path where they don't do cross stage stuff, and conditionally adding cross stage linking if all stages are present is slightly unexpected.